src/wallet/did_wallet/did_info.py

```diff
@@ -0,0 +1,16 @@
+from dataclasses import dataclass
+from typing import List, Optional, Tuple
+
+from src.types.sized_bytes import bytes32
+from src.util.streamable import streamable, Streamable
+from src.wallet.cc_wallet.ccparent import CCParent
+from src.types.program import Program
+
+
+@dataclass(frozen=True)
+@streamable
+class DIDInfo(Streamable):
+    my_did: Optional[bytes]
+    backup_ids: List[bytes]
+    parent_info: List[Tuple[bytes32, Optional[CCParent]]]  # {coin.name(): CCParent}
```

Comment:
I know this is a copy of how CCs do it, but it doesn't really scale well. Suppose you have 10000 disparate coins and you want to spend one. You would load this single field with 10000 items, then iterate through looking for your coin. The data should be stored in the coin table, not the wallet table.